sns/parity_math.c

```diff
@@ -229,16 +229,22 @@ static void dependency_bitmap_prepare(struct m0_sns_ir_block *f_block,
 
 #if ISAL_ENCODE_ENABLED
 /**
- * Converts m0_bufvec to m0_buf and vice versa.
- * @param bvec  - Pointer to buffer vector.
- * @param buf   - Pointer to buffer.
- * @param count - Number of buffers to copy
- * @param dir   - Decides direction of conversion.
- *                True  - Copy data from buffer vector to buffer
- *                False - Copy data from buffer to buffer vector
+ * Converts m0_bufvec to m0_buf.
+ * @param bvec[in]  - Pointer to buffer vector.
```

Comment:
This is an array of pointers to m0_bufvec?

Reply:
Yes. I have updated the function and function header.